src/components/Header/SearchHeader.js

```diff
@@ -52,13 +54,17 @@ const SearchHeader = ({
             <GlobalSelector
               extendRight={true}
               handleChange={handleSelectChange}
+              onIsFocusedChanged={setIsGlobalSelectorFocused}
             />
           </SelectorWrapper>
           {isMobile && (
             <MapToggle
               onClick={() => toggleMobileMenu()}
               isActive={mobileMenuOpen}
             >
+              {!isGlobalSelectorFocused && !isNarrowMobile && (
+                <>See county data&nbsp;&nbsp;</>
```

Comment:
Not sure about "See county data" (since it just opens the state/county map; doesn't show you data directly).  Maybe "Find on Map" or just "Map"?

Reply:
I like "Find on Map" because it mirrors the language of the search bar, make it more obvious that it's an alternative to text search.

![image](https://user-images.githubusercontent.com/1570168/78322931-0b2d9d80-7525-11ea-9da2-d6cccfbc0796.png)